### DISCUSSION TITLE
yml file in bepinex/config folder?

### DISCUSSION THREAD

**Discussion by EifelB** [lloesche/valheim-server-docker#312] [Q&A]
Hello,

Thank you very much for this container. So far I have managed well and almost everything is running. I have problems with CreatureConfig.yml. It should be in the BepinEx in Config directory. 

valheim-server Config file is set to active, but no configuration yaml file was found at /opt/valheim/bepinex/BepInEx/plugins/CreatureConfig.yml.  

The mod searches in the directory " opt/valheim/bepinex/BepInEx/plugins. However, the file is copied to opt/valheim/bepinex/BepInEx/conifg. 

How do I extend the stack so that the file is moved into the right one?

**Comment by lloesche** [comment]
Any file that exists in `/config/bepinex/plugins/` on container start will be copied to the `/opt/valheim/bepinex/BepInEx/plugins/` folder.
Usually that's plugin `.dll` files but any other file will be copied as well.

**Comment by EifelB** [comment]
thx worked :) 
oft ist die denke nicht immer gerde aus :)
